artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/JsonUtil.java

```diff
@@ -338,6 +338,9 @@ public static String truncateString(final String str, final int valueSizeLimit)
    }
 
    public static Object truncate(final Object value, final int valueSizeLimit) {
+      if (value == null) {
+         return "";
```

Comment:
Doesnt necessarily follow to me that the truncation of a null is an empty string. Why not return null? Or throw?

Reply:
This method is used to deal with message data which is sent back to the web console for visual inspection. Given that it didn't handle `null` property values properly means that the web console wasn't actually getting any `null` property values either so it wasn't clear to me that it could actually handle them. I didn't want to fix this NPE only to create another one somewhere else.

Also, there was a precedent for using a blank string for a `null` value already in `org.apache.activemq.artemis.core.message.impl.CoreMessage.TextMessageOpenTypeFactory#getFields`. I decided to carry that behavior forward here.